paparazzi/paparazzi/src/main/java/app/cash/paparazzi/DeviceConfig.kt

```diff
@@ -445,6 +446,62 @@ data class DeviceConfig(
       released = "October 15, 2020"
     )
 
+    // https://android.googlesource.com/platform/tools/base/+/mirror-goog-studio-master-dev/sdklib/src/main/java/com/android/sdklib/devices/wear.xml
+    @JvmField
+    public val WEAR_OS_SMALL_ROUND: DeviceConfig = DeviceConfig(
+      screenHeight = 384,
+      screenWidth = 384,
+      xdpi = 320,
+      ydpi = 320,
+      orientation = ScreenOrientation.PORTRAIT,
+      density = Density.XHIGH,
+      ratio = ScreenRatio.LONG,
+      size = ScreenSize.SMALL,
+      keyboard = Keyboard.NOKEY,
+      touchScreen = TouchScreen.FINGER,
+      keyboardState = KeyboardState.HIDDEN,
+      softButtons = true,
```

Comment:
Given that `<d:buttons>hard</d:buttons>` is in the xml definition, shouldn't `softButtons = false`? This also gets rid of them for you in #525.

Reply:
Thanks!